src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx

```diff
@@ -0,0 +1,143 @@
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Metric
+      </th>
+      <th>
+        Screen
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+      <tr>
+        <td>
+          Response Time
+        </td>
+        <td>
+          All screens
+        </td>
+        <td>
+          The average duration of sampled calls between two services in the initial view, or between transactions in the other views. How well this represents actual system performance depends on the effective sample rate. This metric is displayed on each level as you drill down.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Throughput
+        </td>
+        <td>
+          Opening screen
+        </td>
+        <td>
+          ??
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Error rate
+        </td>
+        <td>
+          Opening screen
+        </td>
+        <td>
+          ??
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Traced call count
+        </td>
+        <td>
+          Drill-down pages
+        </td>
+        <td>
+          Represents the number of sampled calls we have for a given path between two services or transactions. This is lower than the total throughput, unless you are sampling 100% of your requests.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Traced error count
+        </td>
+        <td>
+          Drill-down pages
+        </td>
+        <td>
+          The number of sampled calls between two services or transactions that had errors.
+        </td>
+      </tr>
+  </tbody>
+</table>
+
+## The external services table [#table-view]
+
+As a companion to the map view, the table view lists all the related services in a columnar format. When you click **List** in the upper-right of the page, you see the same services from the map view:
+
+![Screenshot showing the table version of the external services map](./images/table-view.png "Screenshot showing the table version of the external services map")
+
+Similar to the map view, you can click on specific entities (services) to see transactions in the drill-down tables. As you drill down and find an interesting endpoint, you can click on **Traces** to switch over to distributed tracing details.
+
+To the right of each performance metric column is a corresponding **% change** (percentage change) column. The percentage change calculation is based on the time frame you choose in the main time picker and the comparison time picker (**Compare to**). The comparison time picker indicates how much before the main time window the comparison should start. 
+
+For example, with **Response time**, if the current time is 11 a.m. and the main time picker is **last 30 minutes** and the **Compare to** time picker is **1 hour ago**, the duration is the average from 10:30-11:00 a.m., and the **% change** compares that to the average from 9:30-10:00 a.m.
```

Comment:
Maybe turn into bullets?
Here is an example:
With Response time, the current time is 11 a.m. The main time picker is Last 30 minutes and the Compare to time picker is 1 hour ago.
* The duration is the average from 10:30-11:00 a.m.
* The % change compares that to the average from 9:30-10:00 a.m.

Reply:
@barbnewrelic, fabulous suggestion!!